airflow/dags/gtfs_ckan_uploader/calitp_data_ca_exporter.py

```diff
@@ -0,0 +1,66 @@
+import os
+import requests
+from calitp import get_table
+
+
+#
+API_ENDPOINT = "https://data.ca.gov/api/3/action/resource_update"
+API_KEY = os.environ.get("someAPIKey")
+#
+
+agency = get_table("gtfs_schedule.agency", as_df=True)
+agency.to_csv("gtfs_schedule_agency.csv")
```

Comment:
feels like this should be a AIRFLOW secret or connection?

Reply:
(sorry that should be about hte `API_KEY` line

Reply:
If you change `"someAPIKey"` back to the string hunter used in code example in the issue then, we can set that environment variable in the composer UI together.